NeoMathEngine/src/CPU/CpuMathEngineBlas.cpp

```diff
@@ -444,6 +444,7 @@ void CCpuMathEngine::VectorMultichannelLookupAndCopy(int batchSize, int channelC
 		}
 		int remained = channelCount - lookupCount;
 		PRESUME_EXPR(remained == 0);
+		(void)remained; // Avoid unused variable warning when PRESUME_EXPR is empty (non-debug)
```

Comment:
you can modify ASSERT_EXPR at the beginning and remove this check
also you can move other `remained` variables in this file out of scope